en/asgardeo/docs/apis/organization-apis/restapis/org-management.yaml

```diff
@@ -429,6 +429,45 @@ paths:
             curl --location 'https://api.asgardeo.io/t/{organization-name}/o/api/server/v1/organizations/metadata' \
             -H 'Accept: application/json' \
             -H 'Authorization: Bearer {bearer_token}'
+  /organizations/meta-attributes:
+    get:
+      tags:
+        - Organization Meta Attributes
+      description: |
+        This API facilitates the retrieval of organization meta attributes which matches the defined search criteria, if any.
+        
+        Supported operators: "eq"(equals), "co"(contains), "sw"(starts with), "ew"(ends with), "ge"(greater than or equals), "le"(less than or equals), "gt"(greater than), "lt"(less than)
+        
+        Multiple filters can be combined using the "and" operator.
+        
+        Example: filter=attributes+eq+Country
+        
+        <b>Scope(Permission) required:</b> `internal_org_organization_view`
+      summary: Get meta attributes of organizations with filter capabilities.
+      operationId: organizationsMetaAttributesGet
+      parameters:
+        - $ref: '#/components/parameters/filterQueryParam'
+        - $ref: '#/components/parameters/limitQueryParam'
+        - $ref: '#/components/parameters/afterQueryParam'
+        - $ref: '#/components/parameters/beforeQueryParam'
+        - $ref: '#/components/parameters/recursiveQueryParam'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetaAttributesResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/ServerError'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
```

Comment:
We need to add sample curls in the API swaggers used for docs.

Check other endpoints `x-codeSamples` section. Similarly added for the new endpoint

Reply:
This is now added in all places.